app/javascript/packages/time-element/index.ts

```diff
@@ -56,3 +56,9 @@ export class TimeElement extends HTMLElement {
     }
   }
 }
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'lg-time': TimeElement;
```

Comment:
It's a bit non-obvious, but what this is doing is making it so that the `createElement` call in the spec will have the current inferred return type based on [`document.createElement`](https://github.com/18F/identity-idp/blob/f6719103546d324a1eea29bc1061cf2efcff191f/app/javascript/packages/time-element/index.spec.ts#L12) so that [the reference to `element.formatter`](https://github.com/18F/identity-idp/blob/b6c3e8bf72fcd7e2385bc022a16c17a2e2bc4778/app/javascript/packages/time-element/index.spec.ts#L62) doesn't produce a type error.

Before|After
---|---
![image](https://user-images.githubusercontent.com/1779930/186913045-3838c08b-73c7-4609-8a08-ce218950adab.png)|![image](https://user-images.githubusercontent.com/1779930/186912987-0c52441a-0f9f-4ce6-83a6-cbbc33b4bbfe.png)